tests/migrations/test_commands.py

```diff
@@ -723,17 +723,15 @@ def test_makemigrations_interactive_accept(self):
                 self.assertTrue(os.path.exists(merge_file))
             self.assertIn("Created new merge migration", force_text(out.getvalue()))
 
-    def test_makemigrations_default_merge_name(self):
-        test_date = datetime.datetime(2016, 1, 2, 3, 4)
-        test_date_mock = mock.patch('django.utils.timezone.now', mock.Mock(return_value=test_date))
-        with mock.patch('django.db.migrations.questioner.input', mock.Mock(return_value='y')), test_date_mock:
+    @mock.patch('django.db.migrations.utils.datetime')
```

Comment:
The issue is that `@mock.patch('django.db.migrations.utils.datetime.datetime.now')` gives `TypeError: can't set attributes of built-in/extension type 'datetime.datetime'`.